src/dparse/parser.d

```diff
@@ -599,6 +605,9 @@ class Parser
             if (node.identifierOrIntegerOrOpcode == tok!"identifier" && currentIs(tok!":"))
             {
                 advance(); // :
+                node.isLabel = true;
```

Comment:
i added this property because there can be single opcode instructions such as `RDTSCP`.